antarest/study/service.py

```diff
@@ -1559,15 +1561,12 @@ def change_owner(self, study_id: str, owner_id: int) -> None:
             )
         )
 
-        owner_name = None if new_owner is None else new_owner.name
-        self._edit_study_using_command(study=study, url="study/antares/author", data=owner_name)
+        command_context = self.storage_service.variant_study_service.command_factory.command_context
+        args = {"author": new_owner.name, "study_version": study.version, "command_context": command_context}
+        command = ReplaceStudyAuthor.model_validate(args)
+        self.get_study_interface(study).add_commands([command])
```

Comment:
Note that it's very weird (actually buggy I think) that we edit the actual study metadata in database, and the files through a command:
if we later remove the command, there will be a difference between the database and the file.

So after all I am not sure we should enable that.
If we have duplicate data between the database and the files, we should probably only choose one as the source of truth, which here would be the database.
Then, only at export time, we could udpate the files with the actual data.

This is not perfect but it looks more consistent.